### DISCUSSION TITLE
[Feature] not charging all albums in albums page

### DISCUSSION THREAD

**Discussion by githubbab** [immich-app/immich#5260] [Feature Request]
### The feature

I have lot of albums and when i go to the albums page, charging take long time. 
Is it possible to do like Photos and favortites pages and do progressive charging ?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by martabal** [comment]
Great suggestion, however implementing it is considerably more challenging than simply loading all albums.
Out of curiosity, how many albums do you have to experience this loading time ?

**Reply by githubbab** [reply]
I have near than 900 albums and loading is little more than 10s.

**Reply by martabal** [reply]
With the cover mode or the list mode ?

**Reply by githubbab** [reply]
both.
I don't use list mode but after test, loading take same time for the 2 modes

**Reply by githubbab** [reply]
But switch mode is fast.

**Comment by githubbab** [comment]
I have see that is the same problem when i want to put a picture in album.
It seem load all albums before.
